### DISCUSSION TITLE
keda addons not present in list on microk8s 1.23 stable

### DISCUSSION THREAD

**Discussion by GitMax89** [kedacore/keda#2913] [General]
Hello,

i am using microk8s on ubuntu 20.04 virtualized on parallels in my macbook pro M1 Pro.

Why is the Keda addon not present and activated?
Hasn't the functionality been introduced since v2.0?

how can i integrate keda on kubernetes and M1 processor?

Thanks

**Comment by zroubalik** [comment]
Hi, we don't manage microk8s plugins, it might be better to ask in the microk8s community.
